build/PlatformInstaller.targets

```diff
@@ -0,0 +1,298 @@
+      -e 'KOREBUILD_DOTNET_FEED_UNCACHED=$(KOREBUILD_DOTNET_FEED_UNCACHED)'
+      -e &quot;KOREBUILD_DOTNET_FEED_CREDENTIAL=$KOREBUILD_DOTNET_FEED_CREDENTIAL&quot;
+      docker-image-$(Image)
+      ./build.sh /t:RunDebTool"
+      ContinueOnError="WarnAndContinue" />
+
+    <!-- Copy SF and Hosting Bundle packages to output -->
+    <ItemGroup>
+      <GeneratedDebFiles Include="$(_WorkOutputDir)/*.deb" />
+    </ItemGroup>
+
+    <Error Text="@(GeneratedDebFiles->Count()) deb installer files generated." Condition="'@(GeneratedDebFiles->Count())' != 1" />
+
+    <Copy
+      DestinationFiles="$(_InstallersOutputDir)$(DebPrefix)-$(DebVersion)-$(Image)-x64.deb"
+      SourceFiles="@(GeneratedDebFiles)"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+  </Target>
+
+  <Target Name="_GenerateDebOnPlatform">
+    <!-- Build Docker Image -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_BuildDockerImage" Properties="Image=$(Image)" />
+
+    <ItemGroup>
+      <_DebSFDependencies Include="aspnetcore-store-2.0.3"/>
+      <_DebHostingDependencies Include="$(DotnetRuntimeInstallerName)"/>
+      <_DebHostingDependencies Include="$(SFInstallerName)-$(Version)"/>
+    </ItemGroup>
+
+    <PropertyGroup>
+      <DebSFDependencies>@(_DebSFDependencies->'"%(Identity)": {}', ', ')</DebSFDependencies>
+      <DebHostingDependencies>@(_DebHostingDependencies->'"%(Identity)": {}', ', ')</DebHostingDependencies>
+
+      <CommonProps>Image=$(Image);DebVersion=$(Version)</CommonProps>
+      <DebSFProps>DebPrefix=$(SFInstallerName);DebSummary=$(SFSummary);DebDescription=$(SFDescription)</DebSFProps>
+      <DebSFProps>$(DebSFProps);;DebDependencies=$(DebSFDependencies);SFArchive=$(SFLinuxArchiveFilePath)</DebSFProps>
+      <DebHostingProps>DebPrefix=$(HostingInstallerName);DebSummary=$(HostingSummary);DebDescription=$(HostingDescription)</DebHostingProps>
+      <DebHostingProps>$(DebHostingProps);DebDependencies=$(DebHostingDependencies)</DebHostingProps>
+    </PropertyGroup>
+
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebSFProps)" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDeb" Properties="$(CommonProps);$(DebHostingProps)" />
+
+    <!-- Remove Docker Image to save disk space -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_RemoveDockerImage" Properties="Image=$(Image)" />
+  </Target>
+
+  <Target Name="GenerateDebs" DependsOnTargets="_EnsureInstallerPrerequisites">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="Version=$(Version);Image=debian.8" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="Version=$(Version);Image=ubuntu.14.04" />
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="Version=$(Version);Image=ubuntu.16.04" />
```

Comment:
I need to check which platform we need to build for in 2.1. Copying the current platforms minus the 16.10 which we know will no longer be supported.